test/lib/ansible_test/_data/sanity/yamllint/yamllinter.py

```diff
@@ -8,6 +8,12 @@
 import os
 import sys
 
+import yaml
+from yaml.resolver import Resolver
+from yaml.constructor import SafeConstructor
+from yaml.error import MarkedYAMLError
+from _yaml import CParser
```

Comment:
This change should eliminate the pylint error. Unfortunately pylint can't see what's available in C extensions.

```suggestion
from _yaml import CParser  # pylint: disable=no-name-in-module
```